help/md/dnCBDSP.md

```diff
@@ -1,9 +1,38 @@
 ## name
 dnCBDSP
 ## title
+Conditional birth-death shift process
 ## description
+Simulates a tree under a birth-death process with shifts in birth and death rates.
```

Comment:
We should ideally specify whether these are shifts among lineages or through time. In general, we seem to follow a convention of using "shift" for the former and "episodic" for the latter, but users most likely won't know this, so it would be good to be explicit about it.